use-cases/school-timetabling/src/main/java/org/acme/schooltimetabling/service/TimeTableResource.java

```diff
@@ -0,0 +1,165 @@
+package org.acme.schooltimetabling.service;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import ai.timefold.solver.core.api.solver.SolverManager;
+import ai.timefold.solver.core.api.solver.SolverStatus;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.acme.schooltimetabling.domain.TimeTable;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Tag(name = "School Timetables", description = "School timetable service assigning lessons to rooms and timeslots.")
+@Path("timetables")
+public class TimeTableResource {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TimeTableResource.class);
+
+    private final SolverManager<TimeTable, String> solverManager;
+
+    private ConcurrentMap<String, Job> jobs = new ConcurrentHashMap<>();
```

Comment:
// TODO there is not time to live, so this map grows out of memory

Reply:
jobs -> jobMap

Reply:
I think I've found a good naming pattern for maps: `keyToValue`, in this case `jobIdToJob`. Map doesn't say anything the IDE would not say already; this gives you the semantics of the key and value in that exact order.